Assets/LeapMotion/Core/Scripts/Attributes/CurveBounds.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+using UnityEngine;
+
+namespace Leap.Unity.Attributes {
+
+  /// <summary>
+  /// You can use this attribute to mark that an AnimationCurve can only have
+  /// values that fall within specific bounds.  The user will be prevented from
+  /// entering a curve that lies outside of these bounds.
+  /// </summary>
+  public class CurveBounds : CombinablePropertyAttribute, IFullPropertyDrawer {
```

Comment:
Shouldn't Attributes be called e.g. CurveBoundsAttribute?

Reply:
Ah yes!  Always forget about that...